documentation/public-task-api.md

```diff
@@ -33,15 +33,14 @@ We add an infinitely cacheable (`Cache-Control: public, max-age=604800`) endpoin
   "leader": "https://dap.xxqbi.example/",
   "helper": "https://dap.xxqbi.example/",
   "time_precision_seconds": 1080,
-  "protocol": "DAP-04"
```

Comment:
This is removed because we have no way of determining per-task protocol currently. We will likely want to add that eventually